packages/mobile/src/components/rate-cta-drawer/RateCtaReminder.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from 'react'
+
+import { accountSelectors } from '@audius/common'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { useAsyncStorage } from 'app/hooks/useAsyncStorage'
+import useSessionCount from 'app/hooks/useSessionCount'
+import { requestReview } from 'app/store/rate-cta/slice'
+
+import { ASYNC_STORAGE_KEY } from './constants'
```

Comment:
love that you used constants here, i need to do that more. could the name maybe be more related to the rate cta, like RATE_CTA_STORAGE_KEY or smthg?